team site and account deletion (#354)
gristlabs/grist-help: help/en/docs/teams.md

If you created a team site on the Pro or Business plan, or were added to it as a Billing Manager, then you may manage billing information, and edit your team site's name and subdomain from the billing account page. Open the [user menu](glossary.md#user-menu) and click on "Billing Account" to open a menu that looks like this.

> <span class="screenshot-large">*![billing account new image](images/billing-account-new-image.png)*</span>

Author: "always name the new image the same as the old image" ... yes, I realized this as I inputted a new image and deleted the old one. But, had already done so, so didn't want to retrace unnecessary steps. 

In terms of my email from the image... noted. Will make the necessary change!

> <span class="screenshot-large">*![billing account new image](images/billing-account-new-image.png)*</span>

Reviewer: @MichaelNickolas let me know when changes have been made and I'll do a final review.

Click on the top "Change" icon to edit your team site name or subdomain. Note, if you change the
subdomain, then any links pointing to your site or documents within your site will need updating.

> You can also delete a team site under Billing Summary by clicking "Delete team site".

Reviewer 2: Rephrase and make all a single paragraph.

```suggestion
Team sites can be deleted from the Billing Summary page by clicking "Delete team site". Please note that this action cannot be undone and this will remove your site with all documents. Make sure you have created a backup of important data beforehand.
```

Did you test this functionality? What happens if there are team members still assigned to the team, etc?

> 

Reviewer 2: ```suggestion

```

> Please note that this action cannot be undone and this will remove your site with all documents. Make sure you have created a backup of important data beforehand.

Reviewer 2: ```suggestion
```

If you created a team site on the Pro or Business plan, or were added to it as a Billing Manager, then you may manage billing information, and edit your team site's name and subdomain from the billing account page. Open the [user menu](glossary.md#user-menu) and click on "Billing Account" to open a menu that looks like this.

<span class="screenshot-large">*![billing account](images/team-sharing/billing-account.png)*</span>

Click on the top "Change" icon to edit your team site name or subdomain. Note, if you change the
subdomain, then any links pointing to your site or documents within your site will need updating.

> Team sites can be deleted from the Billing Summary page by clicking "Delete team site". Please note that this action cannot be undone and this will remove your site with all documents. Make sure you have created a backup of important data beforehand.

Reviewer: "with all documents" -> "including all documents" is a bit more clear, I think.
